frappe/core/doctype/server_script/server_script.py

```diff
@@ -85,7 +85,7 @@ def execute_method(self) -> Dict:
 			raise frappe.PermissionError
 
 		# output can be stored in flags
-		_globals, _locals = safe_exec(self.script)
+		_globals, _locals = safe_exec(self.script, _globals={"args": kwargs})
```

Comment:
This has been possible so far as well.. with `frappe.form_dict`
Is `args` supposed to replace that? If that is the case, shouldn't `kwargs` also be specified in `frappe.handler.execute_cmd`?

IMO, we could just keep it `frappe.form_dict`, unless there's a drawback to this?

Reply:
Done